### ISSUE TITLE
Unable to switch profiles

### ISSUE THREAD

**Issue by jdblack** [maas/python-libmaas#164]
Trying to switch between multiple maas profiles is failing.  A sqlite error is falling through 👍 

```
# "switching" to a different existing profile breaks
Tue.11:27  jblackwell@work~$ maas profiles switch kafka-sac1
Error: UNIQUE constraint failed: index 'only_one_profile_selected'

```
```
~$ which maas
/usr/local/bin/maas
Tue.11:30  jblackwell@work
~$ head -n 1 /usr/local/bin/maas
#!/usr/local/opt/python3/bin/python3.6
Tue.11:31  jblackwell@work

~$ python3.6 --version
Python 3.6.5
Tue.11:31  jblackwell@work
~$ pip3.6 list | grep maas
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
python-libmaas (0.5.0)

```

**Comment by jdblack** [comment]
~$ cat bin/maasprofile
#!/usr/local/bin/python3

import sqlite3
import os
import sys
from subprocess import call


if len(sys.argv) != 2:
  print("Please pick one of the following profiles to log into")
  call(["maas","profiles","list"])
  sys.exit(1)

profile = sys.argv[1]

print("Switching to profile %s" % profile)

c = sqlite3.connect("%s/.maas.db" % os.environ['HOME'])
c.execute("update profiles set selected=0")
c.execute("update profiles set selected=1 where name=?", [profile])

**Comment by blakerouse** [comment]
I have released 0.6.1 on PyPI please verify this fixes your issue.
